doc/user/content/sql/appendix-cluster-sizes.md

```diff
@@ -7,8 +7,40 @@ menu:
     weight: 900
 ---
 
+## Default Cluster Sizes
+
 {{% self-managed/materialize-cluster-sizes %}}
 
+## Custom Cluster Sizes
```

Comment:
Moved the content into the appendix to lower the prominence since we don't want to encourage people to override the defaults.